docs/slsa.md

```diff
@@ -134,12 +145,15 @@ Describes the config that initialized the build.
       },
 ```
 
-For builds that were initialized from a remote context like Git URL or HTTP URL, this structure
-defines to URL to the context and its immutable digest. For frontend builds, the `entryPoint` field
-defines the path for the Dockerfile that initialized the build (`filename` frontend option).
+For builds that were initialized from a remote context, like a Git or HTTP URL, this object
+defines to URL to the context and its immutable digest. For builds using a local frontend,
+such as a Dockerfile, the `entryPoint` field defines the path for the frontend file that
+initialized the build (`filename` frontend option).
```

Comment:
I think we should explicitly reference the `uri` and `digest` fields like we do for `entryPoint`.